.changeset/shiny-singers-nail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-test-data/cart-discount': minor
```

Comment:
Nitpick: as we are fixing a wrong behavior, I'd expect changes type to be `patch`